interface/portraits/portraits_tauren.gfx

```diff
@@ -13,7 +13,17 @@ spriteTypes = {
 			"GFX_character_background:p0"
```

Comment:
Looks like you compressed it a lot hurting quality.
It's pixelated.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/105497598-817bd600-5cd8-11eb-9df7-c83ce1d2c06d.png)

</details>